### PR TITLE
Updates to privacy field in DATS.json

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR updates the privacy field across subdatasets (except for those generated by crawlers) for compatibility with the standard format described in https://github.com/CONP-PCNO/conp-dataset/issues/536.

### CODE REVIEW

**Comment by PapillonMcGill** [comment]
1000Genome should be "open" not "public"
BigBrain projects (3) should have "adult" in addition to "Homo sapiens"
Brainspan should be "open" not  "public"
celltypes should be  "open" not  "public"
mousebytesdataset should have "Alzheimer" in addition to "Mus musculus"

**Comment by emmetaobrien** [comment]
I've corrected all of those except the last; did we definitely conclude we wanted to record the disease state of a dataset under `isAbout` ?

**Comment by PapillonMcGill** [comment]
That is what the spreadsheet say.

**Comment by emmetaobrien** [comment]
OK, I have updated that also; may want to revisit it later.

**Comment by PapillonMcGill** [comment]
sorry I haven't been clearer, there are also
 BigBrain_3DClassifiedVolumes and 
 BigBrain_3DSurfaces to add  "adult" in addition to "Homo sapiens"

**Comment by cmadjar** [comment]
> I've corrected all of those except the last; did we definitely conclude we wanted to record the disease state of a dataset under isAbout?

@emmetaobrien I also remember that the disease state would be in the isAbout field, in addition to the species studied.

**Comment by emmetaobrien** [comment]
Sorry, forgot to `git add DATS.json` for that last correction.  This should now be ready to merge.

File: .gitmodules [hunk 1]
```diff
@@ -89,7 +89,7 @@
 	url = https://github.com/conp-bot/conp-dataset-Comparing-Perturbation-Modes-for-Evaluating-Instabilities-in-Neuroimaging-Processed-NK.git
 [submodule "projects/multicenter-phantom"]
 	path = projects/multicenter-phantom
-	url = https://github.com/emmetaobrien/multicenter-phantom.git
+	url = https://github.com/conpdatasets/multicenter-phantom.git
```
Comment (cmadjar) [inline .gitmodules:92]: @emmetaobrien Nice! You were able to resolve the git-annex branch conflicts in the end with conpdatasets?

**Review by cmadjar — CHANGES_REQUESTED** [review]
@emmetaobrien Thank you for this!

Two comments:
- for the multicenter phantom dataset, I remember there were some conflicts between your repo and the conpdatasets repo on the git-annex branch. Were you able to resolve those? Just checking.
- in mousebytes, should be `Alzheimer's disease` instead of `Alzheimer's Disease`.

**Review by PapillonMcGill — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (emmetaobrien) [reply .gitmodules:92]: I was able to resolve those conflicts, yes.  I am correcting the `mousebytes` field now.

**Review by cmadjar — CHANGES_REQUESTED** [review]
@emmetaobrien I am confused by your last commit. It removes completely the multicenter-phantom dataset. Instead, shouldn't it link it to your fork of the multicenter-phantom dataset?

**Comment by emmetaobrien** [comment]
I am trying to solve this problem by removing the submodule and reinstalling; you've just caught me in the middle of the process.

**Review by cmadjar — APPROVED** [review]
LGTM

**Review by cmadjar — CHANGES_REQUESTED** [review]
download of multicenter-phantom appears to be failing even though the tests did not pick that up...

**Review by cmadjar — APPROVED** [review]
LGTM.

**Pull request merged**